### PR TITLE
Adds labels to the desciription list row.

### PR DESCRIPTION
**Pull request by adkron**
We would like to be able to have a field and the label be different
sometimes.

### CODE REVIEW

**Comment by adkron** [comment]
I didn't write a frontend test. I was really wanting to add the docs, and got over excited.

File: lib/components/description_list.ex [hunk 1]
```diff
@@ -21,7 +30,7 @@ defmodule Lilly.Components.DescriptionList do
         <div class="border-t border-gray-200">
           {#for {row, is_striped} <- Enum.zip(@rows, Stream.cycle([false, @striped]))}
             <div class={"px-4 py-5 sm:grid sm:grid-cols-3 sm:gap-4 sm:px-6", "bg-gray-300": is_striped}>
-              <dt class={"text-sm font-medium text-gray-500", row.class}>{row.field}</dt>
+              <dt class={"text-sm font-medium text-gray-500", row.class}>{row.label || row.field}</dt>
```
Comment (401matthall) [inline lib/components/description_list.ex:33]: And `label` takes precedence here, right?

**Comment by 401matthall** [comment]
Nice documentation! I know we talked about this and I still forgot about it before merging the previous branch.

**Review by 401matthall — APPROVED** [review]
Nice documentation!

**Pull request merged**